pkg/interface/service/currency-service.go

```diff
@@ -0,0 +1,9 @@
+package service
+
+import "genesis-currency-api/pkg/dto"
+
+type CurrencyService interface {
```

Comment:
https://go.dev/wiki/CodeReviewComments#interfaces

Go interfaces generally belong in the package that uses values of the interface type, not the package that implements those values

Reply:
Done.